SimpleWAWS/Controllers/ResourceController.cs

```diff
@@ -200,19 +209,18 @@ public Task<HttpResponseMessage> All()
                         resourceGroup = await resourceManager.ActivateLogicApp(template as LogicTemplate, identity, anonymousUserName);
                         break;
                     case AppService.Function:
-                        if (identity.Issuer == "OrgId")
-                        {
-                            return Request.CreateErrorResponse(HttpStatusCode.BadRequest, Resources.Server.Error_OrgIdNotSupported);
-                        }
-                        else if (identity.Issuer != "MSA")
-                        {
-                            return SecurityManager.RedirectToAAD(template.CreateQueryString());
-                        }
+                        //if (identity.Issuer == "OrgId")
+                        //{
+                        //    return Request.CreateErrorResponse(HttpStatusCode.BadRequest, Resources.Server.Error_OrgIdNotSupported);
+                        //}
+                        //else if (identity.Issuer != "MSA")
+                        //{
+                        //    return SecurityManager.RedirectToAAD(template.CreateQueryString());
+                        //}
                         resourceGroup = await resourceManager.ActivateFunctionApp(template as FunctionTemplate, identity, anonymousUserName);
                         break;
                 }
-
-                return Request.CreateResponse(HttpStatusCode.OK, resourceGroup == null ? null : resourceGroup.UIResource);
+                return Request.CreateResponse(HttpStatusCode.OK, resourceGroup == null ? null :  (HttpContext.Current.Request.QueryString["appServiceName"] == "Function") ? resourceGroup.FunctionsUIResource : resourceGroup.UIResource);
```

Comment:
When called from try functions it needs to TAS needs to return more data. i.e `FunctionsUIResource` instead of the regular `UIResource`